### PR TITLE
Oxford Nanopore Fast5 files

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    'files': ['include/fast5.hpp'],
+    'dirs': ['include'],
+}
+
+modextrapaths = {
+    'LD_LIBRARY_PATH': 'include',
+}
+
+moduleclass = 'lib'
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb]: @fizwit the style error is the blank line at the end of file 

(you can check it yourself with `--check-contrib`, https://easybuild.readthedocs.io/en/latest/Contributing.html#code-style-review)

File: easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+postinstallcmds = ['mv %(installdir)s/src %(installdir)s/include']
+
+sanity_check_paths = {
+    'files': ['include/fast5.hpp'],
+    'dirs': ['include'],
+}
+
+modextrapaths = {
+    'LD_LIBRARY_PATH': 'include',
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb]: isn't this a header only library?

Comment (fizwit) [reply easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb]: Yes, header only.  How does g++ find this header files if they are not in LD_LIBRARY_PATH?

Comment (migueldiascosta) [reply easybuild/easyconfigs/f/fast5/fast5-0.6.5.eb]: `LD_LIBRARY_PATH` is where the linker looks for libraries, for the compiler to find headers you can add the `include` folder to `CPATH`, or  `CPLUS_INCLUDE_PATH`

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/f7f67f3a1f8d531ad76cd88e0dd118f5 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @fizwit!

**Pull request merged**